### PR TITLE
Update .NET namespaces for AWS

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**

### CODE REVIEW

File: resources.go [hunk 1]
```diff
@@ -2209,6 +2209,55 @@ func Provider() tfbridge.ProviderInfo {
...
+				"codepipeline":      "CodePipeline",
+				"datapipeline":      "DataPipeline",
+				"datasync":          "DataSync",
+				"devicefarm":        "DeviceFarm",
+				"directoryservice":  "DirectoryService",
+				"docdb":             "DocDB",
+				"dynamodb":          "DynamoDB",
+				"directconnect":     "DirectConnect",
+				"ec2clientvpn":      "Ec2ClientVpn",
+				"ec2transitgateway": "Ec2TransitGateway",
```
Comment (CyrusNajmabadi) [inline resources.go]: Ec2 or EC2.  .net generally says "if it's two letters capitalize both.  if its 3+ then title case". i don't know where EC2 lands.

File: resources.go [hunk 2]
```diff
@@ -2209,6 +2209,55 @@ func Provider() tfbridge.ProviderInfo {
...
+				"resourcegroups":    "ResourceGroups",
+				"securityhub":       "SecurityHub",
+				"secretsmanager":    "SecretsManager",
+				"servicecatalog":    "ServiceCatalog",
+				"servicediscovery":  "ServiceDiscovery",
+				"servicequotas":     "ServiceQuotas",
+				"simpledb":          "SimpleDB",
+				"storagegateway":    "StorageGateway",
+				"wafregional":       "WafRegional",
+				"worklink":          "WorkLink",
```
Comment (CyrusNajmabadi) [inline resources.go]: these are the list of names that were clearly wrong.  but there are open questions about names like `emr`.  Should we make that ElasticMapReduce?  Or should we make it Emr.  The latter corresponds to JS/Python.  The former is more .NET-y.

Comment (CyrusNajmabadi) [reply resources.go]: Probably the best thing to do here would be to follow the AWS .net sdk's for consistency with them.  What do you think @lukehoban @pgavlin @mikhail

**Comment by mikhail** [comment]
Love it. Do it

**Pull request merged**